README.md

```diff
@@ -1,60 +1,82 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # js_capstone_project
 
-> 
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-![screenshot](./)
+> In this project we can:
+- retrieve elements from the database with the get method. 
+- retrieve each element with its details with the method get(Id)
+- view the elements with render. 
+- Index.html contains the div for each element I have just selected in my Show.js file in the Display folder. 
 
 
+
+![](https://github.com/vic778/js_capstone_project/blob/Display/img/screen1.png)
 ## Built With
 
-- Webpack
-- JavaScript (ES6)
-- CSS3
 - HTML5
-
+- CSS3 (Flex and media queries)
+- JAVASCRIPT
+- WEBPACK
+- BOOTSTRAP
+- SASS
+- API
 ## Live Demo
-[Click here to view]()
 
-## Getting Started
+[Live Demo Link]()
 
-To get a local copy up and running follow these simple steps.
+## Author 1
 
-### Prerequisites
+👤 **Victor Barh**
 
-some knowledge on `npm` webpack and ES6
+- GitHub: [@Vvic778](https://github.com/vic778)
+- Twitter: [@victoirBarh](https://twitter.com/)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/victoir-barh)
 
-### Setup
+## Getting Started
 
-- Clone repository using this command in terminal `https://github.com/vic778/js_capstone_project.git`
-- Change directory to the cloned repository `cd js_capstone_project`
-- Install dependencies using `npm install`
-- run `npm start`
-### Install
+### Prerequisites:
 
-- Node modules
-- HtmlWebpackPlugin `npm install --save-dev html-webpack-plugin`
-- Load CSS `npm install --save-dev style-loader css-loader`
-- Setup local dev server `npm install --save-dev webpack-dev-server`
+- Web browser
+- Code Editor
+- Live Server Extension
 
+### Cloning the repo to your local system (If you already have git, installed in your system):
 
+- [Copy this link](https://github.com/Buyaki01/js_capstone_project/tree/Get-AllItem)
+- Open your terminal or command line
+- Run "git clone [Paste this link](https://github.com/Buyaki01/js_capstone_project/tree/Get-AllItem)"
+- Open the folder with your code editor
+- Now You can edit the code and check the changes in the browser using Live Server
```

Comment:
I suggest you add the commands so that any recruiter who doesn't have a technical background can easily run the project locally.

For instance: 
> git clone https://github.com/vic778/js_capstone_project.git

cd js_capstone_project

and then npm i and npm start